nuc_morph_analysis/lib/preprocessing/neighbor_analysis/neighbor_analysis.py

```diff
@@ -88,7 +88,7 @@ def compute_neigh(tup):
             return return_dict
 
 
-def compute_density(df, global_df, num_workers=1):
+def compute_density(df, global_df, num_workers=1, old_unfiltered_method=False):
```

Comment:
Same suggestion to rename this flag to not reference the "old method" but just a filter flag

Reply:
or dont even leave the option... its in the git history